demos/common/models/src/openpose_decoder.cpp

```diff
@@ -1,16 +1,27 @@
-// Copyright (C) 2018-2019 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
+/*
+// Copyright (C) 2018-2020 Intel Corporation
 //
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
 
 #include <algorithm>
 #include <utility>
 #include <vector>
 
 #include <samples/common.hpp>
+#include "models/openpose_decoder.h"
 
-#include "peak.hpp"
 
-namespace human_pose_estimation {
 Peak::Peak(const int id, const cv::Point2f& pos, const float score)
```

Comment:
Does it make sense to write a G-API kernel for finding peaks?

Reply:
potentially yes, but this will require series of experiments, so better to have a separate PR for such work

Reply:
I agree. It is certainly better to keep it in separate PR. But, thanks for the confirmation!